### ISSUE TITLE
Feature request: make schema generate more verbose.

### ISSUE THREAD

**Issue by gmos** [piccolo-orm/piccolo#447]
It would be quite helpful when schema generate would let us know on which table and column it was working when it bails out.

**Comment by dantownsend** [comment]
It runs a bunch of coroutines at the same time (one for each table in the database):

https://github.com/piccolo-orm/piccolo/blob/5aea0fdc4f40a99615f534ec49a38b242a5f5cb0/piccolo/apps/schema/commands/generate.py#L816

Originally it was sequential, but it took too long.

It could be modified so it's `asyncio.gather(*table_coroutines, return_exceptions=True)`. We can then work out which ones failed, and tell the user.

**Comment by AliSayyah** [comment]
@dantownsend Hey there!
I would like to work on this, if possible.

**Comment by gmos** [comment]
You already get the exception text printed.  Perhaps as a first step just put the table name in the text of the exceptions that Piccolo explicitly raises.

**Comment by dantownsend** [comment]
@AliSayyah That would be great, thanks.

@gmos OK, makes sense.

**Comment by AliSayyah** [comment]
@dantownsend Should we actually raise the exceptions gathered, or just handle it like warnings are handled?
maybe we can add an `exception` block?

**Comment by dantownsend** [comment]
@AliSayyah Things get tricky with `asyncio.gather` with `return_exceptions=True`, because you can end up with 10 exceptions returned.

We could print out each exception, and then raise a single generic exception at the end like `Exception('Schema generation failed')`. Whatever you think is best.

**Comment by AliSayyah** [comment]
> @AliSayyah Things get tricky with `asyncio.gather` with `return_exceptions=True`, because you can end up with 10 exceptions returned.
> 
> We could print out each exception, and then raise a single generic exception at the end like `Exception('Schema generation failed')`. Whatever you think is best.

Yeah, because of that, I raised a single exception that includes all the exceptions gathered with `asyncio.gather` .

```python
raise Exception([Exception("Error"), ...])
```

**Comment by sinisaos** [comment]
Completed in #449
